add brief section to readme about building docker images

chazu opened the pull request:
Per @sochix suggestion in #247 - This PR provides a short section in the README on how to build the docker images. It doesn't lay out the specifics for each image, but I can add more detail if you like.

Let me know what other changes you prefer before merge, or feel free to tweak the doc.

Thanks!

chazu committed:
add brief section to readme about building docker images

sochix commented:
@chazu can you add instruction on how to modify docker-compose to use self-built images?

chazu committed:
Add instructions on building with docker-compose

Adds instructions on building images via their inclusion in a `docker-compose.yml` and adds clarification on images which require building locally before docker images will build.

sochix commented:
Awesome!
